readthedocs/rtd_tests/utils.py

```diff
@@ -12,10 +12,24 @@
 from django_dynamic_fixture import new
 from django.contrib.auth.models import User
 
+from readthedocs.doc_builder.base import restoring_chdir
 
 log = logging.getLogger(__name__)
 
 
+def get_readthedocs_app_path():
+    """
+    Return the absolute path of the ``readthedocs`` app.
+    """
+    path = getcwd()
+    if path.endswith('readthedocsinc'):
+        # readthedocs-corporate
+        path = pjoin(path, '..', '..', 'readthedocs.org', 'readthedocs')
```

Comment:
Also, -1 on this pattern in general. We need to stop making assumptions that these two repos are siblings in a similar path.